### DISCUSSION TITLE
Question about TDS.init()

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#289] [Internal Design] [answered]
Hi Hantao,

I noticed that when using TDS.init(), it will return an array as the function finally return `system.dae.xy`. As a result, when using `init` function to test the initialization, the output will be large.

I'm wondering if that is not necessary we can return a bool value?

Regards,
Jinning

**Comment by jinningwang** [comment]
As I further checked, the function is not used for variable evaluation, I think it is safe and useful to revise the return value?

**Comment by cuihantao** [comment, marked as answer]
To suppress the output in IPython or Jupyter, you can assign the return to a random variable, like

xy = ss.TDS.init()

Regards,
Hantao Cui

**Reply by jinningwang** [reply]
Thanks! This is a more convenient solution.
